Update index.md (#56)
uk-tre/website: docs/structure/sde-tre-terminology.md

# SDE/TRE Terminology

## Charter

> This is the main document to refer to, it should contain all of the below and further detail: https://www.dropbox.com/scl/fo/fvttq0l9a8i1wfqrcgsdp/AKIK5j3JuXBIM-BFqY6cEAI?rlkey=oke7qkrr8b0vanabs6bg1sq46&dl=0 

Author: ```suggestion
This is the main document to refer to, it should contain all of the below and further detail: [SDE/TRE Terminology charter](https://www.dropbox.com/scl/fo/fvttq0l9a8i1wfqrcgsdp/AKIK5j3JuXBIM-BFqY6cEAI?rlkey=oke7qkrr8b0vanabs6bg1sq46&dl=0)
```

> This is the main document to refer to, it should contain all of the below and further detail: [SDE/TRE Terminology charter](https://www.dropbox.com/scl/fo/fvttq0l9a8i1wfqrcgsdp/AKIK5j3JuXBIM-BFqY6cEAI?rlkey=oke7qkrr8b0vanabs6bg1sq46&dl=0)

Author: ```suggestion

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)
```

# SDE/TRE Terminology

## Charter

> 

Author: ```suggestion
```

This is the main document to refer to, it should contain all of the below and further detail: you can find it within this [summary document with information of all working groups](https://docs.google.com/document/d/1uvqALeVixK6PqdkzLFVwDVhc0fPUuBdddyfQzZsRtm4/edit?usp=sharing).

(sorry for the lack of a direct link, but it lives in dropbox)

## Overview

> The SDE / TRE Definitions Working group is focused on clarifying the terms used WITHIN the UK TRE community especially when different parts are performed by different organisations.
> SDEs and TREs are different, but views on how and the extent to which they differ are varied. 

Author: ```suggestion
The SDE / TRE Definitions Working group is focused on clarifying the terms used WITHIN the UK TRE community especially when different parts are performed by different organisations.
SDEs and TREs are different, but views on how and the extent to which they differ are varied.
```

## Co-chairs and contact

> **Co-chair:**  This is Madalyn Hardaker, Kings University, London madalyn.hardaker@kcl.ac.uk
> 
> **Co-chair:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk
> 
> **Contact:**  This is Pete Barnsley, Francis Crick Institute, London  peter.barnsley@crick.ac.uk
> 

Author: ```suggestion
**Co-chair:** This is Madalyn Hardaker, Kings University, London madalyn.hardaker@kcl.ac.uk

**Co-chair:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk

**Contact:** This is Pete Barnsley, Francis Crick Institute, London peter.barnsley@crick.ac.uk

```

## How to get involved

> (mailing list, slack channel, repositories…)

Author: ```suggestion

(mailing list, slack channel, repositories…)
```

Slack: Join #wg-sde-tre-definitions in [Slack](https://join.slack.com/t/uktrecommunity/shared_invite/zt-2gep86apc-QMLyIdrC2oIIsxTRzLxUqA)

> 

Author: ```suggestion
```
